spec/services/news_api_service_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe NewsApiService, :vcr do 
+  describe 'class methods' do 
+    it 'returns an article based on stock' do 
+      response = NewsApiService.search_articles('audi')
+      expect(response).to be_a Hash
+      expect(response).to have_key :articles
+      expect(response[:articles]).to be_a Array
```

Comment:
Out of curiosity, are there be more than one element within the array? 
Perhaps a count to explicitly convey the number of elements would be useful.

Reply:
good idea - edited the test name and added count to test